src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/script.js

```diff
@@ -137,14 +137,14 @@ document.addEventListener("DOMContentLoaded", function(e) {
     window.addEventListener("hashchange", function(e) {
         history.replaceState(contentDiv.scrollTop, document.title);
     });
+    var timeoutId;
     contentDiv.addEventListener("scroll", function(e) {
-        var timeoutID;
-        if (!timeoutID) {
```

Comment:
I don't understand JavaScript, so could you briefly explain how that _used_ to work? It reads like we declare a variable and then immediately check it for being set?

Reply:
Oops, I didn't even spot that. I guess that explains why some browsers were complaining.